### PR TITLE
Autocomplete improvements

### PR DESCRIPTION
**Pull request by oleq**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

The `AutocompleteView` should react to its `#filteredView`'s `execute` event. Moved the history icon from ckeditor5-revision-history to ckeditor5-core. `TextareaView` should fire an event whenever it calculates its height.

### CODE REVIEW

File: packages/ckeditor5-theme-lark/theme/ckeditor5-ui/components/autocomplete/autocomplete.css [hunk 1]
```diff
@@ -20,11 +20,13 @@
 		&.ck-search__results_n {
 			border-bottom-left-radius: 0;
 			border-bottom-right-radius: 0;
+			margin-bottom: -1px;
```
Comment (DawidKossowski) [inline packages/ckeditor5-theme-lark/theme/ckeditor5-ui/components/autocomplete/autocomplete.css:23]: Why do we need it?

**Review by DawidKossowski — APPROVED** [review]
Generally, it looks good

File: packages/ckeditor5-ui/src/textarea/textareaview.ts [hunk 2]
```diff
@@ -193,3 +195,13 @@ function getTextareaElementClone( element: HTMLTextAreaElement, value: string ):
 
 	return clone;
 }
+
+/**
+ * Fired when the textarea view is resized.
```
Comment (DawidKossowski) [inline packages/ckeditor5-ui/src/textarea/textareaview.ts:200]: I'm wondering if `resized` is causing any confusion. In the context of textarea, `resize` is typically used when a user has changed the size.

File: packages/ckeditor5-theme-lark/theme/ckeditor5-ui/components/autocomplete/autocomplete.css (hunk 1 above)
Comment (oleq) [reply packages/ckeditor5-theme-lark/theme/ckeditor5-ui/components/autocomplete/autocomplete.css:23]: When the results panel is displayed next to the input via `bottom: 100%;` or `top: 100%`, you get a double border: one pixel from the input, another from the panel. This mitigates the duplication. We don't have a custom prop for border-width, so I cannot use `calc( -1 * var(--some-custom-prop ) )` here.

**Pull request merged**